app/models/user.py

```diff
@@ -2,13 +2,31 @@
 from werkzeug.security import generate_password_hash, check_password_hash
 from flask_login import UserMixin
 
+
 class User(db.Model, UserMixin):
   __tablename__ = 'users'
 
   id = db.Column(db.Integer, primary_key = True)
   username = db.Column(db.String(40), nullable = False, unique = True)
   email = db.Column(db.String(255), nullable = False, unique = True)
-  hashed_password = db.Column(db.String(255), nullable = False)
+  hashedPassword = db.Column(db.String(255), nullable = False)
+  profilePhoto = db.Column(db.String(255), default="https://bellfund.ca/wp-content/uploads/2018/03/demo-user.jpg")
+  headline = db.Column(db.String(100))
+  bio = db.Column(db.String(255))
+  firstName = db.Column(db.String(80), nullable = False)
+  lastName = db.Column(db.String(80), nullable = False)
+  messages = db.relationship("Message")
```

Comment:
`sentMessages = db.relationship("Message", foreign_keys =["Message.senderId"])`
`receivedMessages = db.relationship("Message", foreign_keys=["Message.recipientId"])`

Reply:
I don't think you need messages as a relationship on your user, because you are more likely to access your messages from your Message model ultimately. We can talk about this more if you'd like!